sympy/printing/pycode.py

```diff
@@ -106,6 +106,12 @@ def __init__(self, settings=None):
         self.known_constants = dict(self._kc, **(settings or {}).get(
             'user_constants', {}))
 
+    def _print_not_supported(self, expr):
```

Comment:
Any reason to not do this on CodePrinter?

Reply:
Not really, changed this just now.